db/global.go

```diff
@@ -0,0 +1,79 @@
+package db
+
+// NewGlobal constructs a new global object.
+func NewGlobal() *Global {
+	return &Global{}
+}
+
+// Path returns the path to the global configuration
+func (g *Global) Path() (string, error) {
+	return rootGlobal, nil
+}
+
+// Prefix returns the path to the global configuration
+func (g *Global) Prefix() string {
+	return ""
+}
+
+// Validate validates the global configuration.
+func (g *Global) Validate() error {
```

Comment:
Sanitize would be better in this case but the interface prescribes validate. I'm not sure this is really the best place for it, but to ensure the values do get set this is the simplest choice.

Maybe moving this to a hook in v2 would be best.